runtime/gc_glue_java/GlobalCollectorDelegate.cpp

```diff
@@ -97,7 +98,16 @@ MM_GlobalCollectorDelegate::initialize(MM_EnvironmentBase *env, MM_GlobalCollect
 
 	/* Balanced and realtime polices will instantiate their own access barrier */
 	if (_extensions->isStandardGC()) {
-		_extensions->accessBarrier = MM_StandardAccessBarrier::newInstance(env);
+
+#if defined(OMR_ENV_DATA64) && !defined(OMR_GC_COMPRESSED_POINTERS)
+		if (1 == (1 & _extensions->fvtest_enableReadBarrierVerification)) {
```

Comment:
if this flag has value 1 set only, so there is no need to mask it. It would make sense if this field might have different different bit pattern set but looks like this is not a case